webapp/store/logic.py

```diff
@@ -154,7 +154,7 @@ def convert_date(date_to_convert):
     :returns: Readable date
     """
     date_parsed = parser.parse(date_to_convert)
-    return date_parsed.strftime("%b %d %Y")
+    return date_parsed.strftime("%B %d %Y")
```

Comment:
This should be:

```suggestion
    return date_parsed.strftime("%d %B %Y")
```

https://wiki.canonical.com/Marketing/Style_guide#Dates

> Single day: 1 January 2013